composer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "wordpress/twentytwenty",
+  "description": "Twenty Twenty is designed with flexibility at its core.",
+  "type": "package",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "WordPress",
+    "Themes"
+  ],
+  "homepage": "https://github.com/WordPress/twentytwenty",
+  "authors": [
+    {
+      "name": "Contributors",
+      "homepage": "https://github.com/WordPress/twentytwenty/graphs/contributors"
+    }
+  ],
+  "require": {
+    "php": ">=5.6"
+  },
+  "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+    "jakub-onderka/php-parallel-lint": "^1.0",
+    "wptrt/wpthemereview": "^0.2.0"
+  },
+  "scripts": {
+    "lint": "@php ./vendor/bin/parallel-lint --exclude app --exclude vendor .",
```

Comment:
Pro-tip: add `--exclude .git` to this.

I've done some testing with this application and while it won't find PHP files in the `.git` directory, if not explicitly excluded, it will still scan the directory, making it the scan slower.

Also: what's the `--exclude app` for ?

Reply:
Oh must be a copy-paste error 😬 I'll fix this and add the `.git` folder :+1: